### ISSUE TITLE
TWE: Research: Checklist for Rapid Qualitative Analysis (RQA)

### ISSUE THREAD

**Issue by dialectic51** [hackforla/internship#338]
### Dependency
- #324 transcription needs to be de-identified

### Overview
We need to develop a step-by-step checklist for conducting Rapid Qualitative Analysis (RQA) at HackforLA. This checklist will be used to reanalyze data from the RP001: IS21 Internship Exit Interview. This checklist can also be used as the basis for developing a Template for conducting RQA and the starting point for creating a Guide.

### Action Items

- [x] Review information collected in Issue # 330 (link in Resources)
  - [x] Review Issue # 335 as an example of a more elaborate checklist, but one that can still serve as a model for this one
- [x] Based on information collected in Issue # 330, write a comment to this issue that provides a step-by-step checklist for conducting RQA
- [x] Link checklist to the resources/instructions sections below 
- [x] Have ~Linda~ Angela and Dewey review the checklist and provide feedback
  - [x] Make any needed changes to checklist based on Linda's feedback
- [x] Create a template of the Summary Template
  - [x] Save template of Summary Template to [google drive folder](https://drive.google.com/drive/folders/1hKEeL17224X_HHPe19D1go6-AEajr9Ij)
  - [x] Place link to template of Summary Template in Resources/Instructions 
- [x] Create a template of the Matrix
  - [x] Save template of Matrix to [google drive folder](https://drive.google.com/drive/folders/1hKEeL17224X_HHPe19D1go6-AEajr9Ij)
  - [x] Place link to template of Matrix in Resources/Instructions 
- [x] Share updated checklist with ~~lead analyst (Angela D.)~~ Dewey Tran for review
- [ ] Create a new issue titled below
   - [ ]  TWE: Research: RP001: Data: Reanalyze
   - [ ] Add links the same links to Resources/Instructions that are listed on this issue
   - [ ] Place in New Issue Approval column 
- [ ] Create a new issue in [UI/UX Templates and Guides](https://github.com/hackforla/UI-UX/projects/1) titled 'Create a Guide/Template: Rapid Qualitative Testing'
   - [ ] Add the same links to Resources/Instructions that are listed on this issue
     - [ ] Remember that a document has [already been created for the guide](https://docs.google.com/document/d/1RjR7Oh9KZ2bQlJGy1MyNFwYjtxeDxawza9P6ViKOL64/edit)
   - [ ] Place in New Issue Approval column 

### Resources/Instructions

1. [Wiki Page: RP001](https://github.com/hackforla/internship/wiki/Research-Plan-1:-IS21-Interns)
1. #330 
1. [Folder on UI/UX CoP Google Drive for Rapid Qualitative Analysis](https://drive.google.com/drive/folders/1d9xZF6MR5OXSrfX8d-MI1BVSZHxesg-A)
1. [HfLA Template on Making a Guide](https://docs.google.com/document/d/1kV29V-mRPBNRRfIbYHrz05FElC02Dh6egaMANlxaZrM/edit#heading=h.c8tc614ce3iu)
1. [UI/UX Guide for How to Conduct Rapid Qualitative Analysis](https://docs.google.com/document/d/1RjR7Oh9KZ2bQlJGy1MyNFwYjtxeDxawza9P6ViKOL64/edit?usp=share_link) 
 1. [Linda's Presentation to UXR Team on How to Conduct Rapid Qualitative Analysis](https://docs.google.com/presentation/d/12j99CF8fnvdb7WOWxDn1_CO_UODjoL23HXZUqujbveA/edit?usp=sharing) 
 2. [Transcript of Hamilton's Presentation on RQA](https://www.hsrd.research.va.gov/for_researchers/cyber_seminars/catalog/transcripts/780.doc)
 1. [Free Computer Assisted Qualitative Data AnalysiS (CAQDAS) programs](https://github.com/hackforla/internship/issues/330#issuecomment-1318876272)
 4. [Potential domain names and categories for Internship RP01](https://github.com/hackforla/internship/issues/338#issuecomment-1325506436)
 5. [Checklist for conducting Rapid Qualitative Analysis (RQA)](https://github.com/hackforla/internship/issues/338#issuecomment-1327682586)
 6. [Template for Summary Template for RQA](https://docs.google.com/document/d/19nwabYmVlpG0GKROYvx5ZVR7R8vKHA--1kmLWACFKWY/edit)
 7. [Template for Matrix Analysis for RQA](https://docs.google.com/spreadsheets/d/1WT4zQXFvST1UygloeXcwa-AVm6x2o5c2U9Nky7Bd8qc/edit#gid=0)

**Comment by dialectic51** [comment]
Here are potential domain names and categorization for the interview questions for [Internship RP01.](https://github.com/hackforla/internship/wiki/Research-Plan-1:-IS21-Interns) They can be used once the lead analyst creates a summary template for RQA (see the checklist created for this issue). The domain names and categorization are based upon the advice provided by Linda in her [presentation on RQA.](https://docs.google.com/presentation/d/12j99CF8fnvdb7WOWxDn1_CO_UODjoL23HXZUqujbveA/edit)

### Domains

Role and responsibilities

- 1 Describe your role and contribution.

Learned skills and discovery
- 2 What softwares or technical skills did you learn or practice?
- 10 Did you have an ‘aha’ moment at Hack for LA? If so, when?

Problem-solving
- 3 What did you do when you encountered a problem in regards to your knowledge of the subject matter?
- 7 How did you mitigate issues like unavailability of your peers and mentors?

Engagement and accountability
- 6 How did you keep your peers informed about your progress/issues?
- 8 How did you document your work?

Social networking
- 9 How did you extend or build your network at Hack for LA?

Pain points
- 11 What was missing or disappointing - how would you improve it? 

Choice of HfLA
- 12 Were you offered more than one internship? Why did you choose our program over other options? 

Prior expectations
- 14 What were your expectations with the internship program at Hack for LA before you applied? After the AMA? After the onboarding?

Recommend, endorse, volunteer
- 15 Would you recommend the internship program at hack for LA to your peers? 
- 17 If time was not a constraint, would you like to be part of the community going forward?

Don’t require qualitative analysis
- 4 How often did you get to interact with your peers and mentors?
  - Could also be placed in “engagement and accountability” domain
- 5 How regularly did you attend the weekly meetings?
  - Could also be placed in “engagement and accountability” domain
  - (Could potentially code this—but unclear if this qualitative analysis would be that useful, but some pain points are identified, for example)
- 13 Which of the hack for LA volunteers was instrumental to your success at hack for LA?
- 16 Which feature did you learn from and enjoy the most? One-on-one, LinkedIn workshop, Standup, team meets.

**Comment by dialectic51** [comment]
### Meta Note

This is a draft of the action items that I will be continuing to add to. I will remove this note once it is complete. 

### Overview

We need to conduct Rapid Qualitative Analysis (RQA) of interview transcripts or other qualitative materials for RP[Replace 000]

### Action Items

**Comment by dialectic51** [comment]
@bradyse I wrapped up [issue #330](https://github.com/hackforla/internship/issues/330) and created this (issue #338) in order to put together the checklist for conducting rapid qualitative analysis. I have completed the checklist [in this comment](https://github.com/hackforla/internship/issues/338#issuecomment-1327682586) in this issue. 

When you have time, could you take a look and make sure I'm doing what we discussed at last week's meeting? If it looks like I'm on the right track, then I will contact Linda, have her review the checklist, and then make changes based on her feedback. We can also discuss this at our meeting tomorrow.

**Comment by bradyse** [comment]
@dialectic51 
This looks good. Go ahead and send the comment link to Linda. Ask her to quickly look through it and determine if the basic steps are there. If there are any glaring errors, she can point those out. Otherwise, I wouldn't expect her to spend more than 20 minutes or so reading through the whole thing.

**Comment by dialectic51** [comment]
@lkawentel 
I have completed a checklist for conducting Rapid Qualitative Analysis (RQA), which I have placed [in the comment above.](https://github.com/hackforla/internship/issues/338#issuecomment-1327682586) I have based the checklist on your answers to Sara's questions, your presentation, and the materials your presentation links to. 

When you have time, please review the checklist and identify any errors or things that need to be added to the list. Sara says this shouldn't take more than 20 minutes or so.

(I have also put together a [list of potential domains based upon your presentation.](https://github.com/hackforla/internship/issues/338#issuecomment-1325506436) I'm going to continue to revise them as I gain more expertise in RQA, but feel free to look them over if you have the time).

**Comment by dialectic51** [comment]
@bradyse 

I have put in a request (via github and slack) for Linda to review the checklist.

I also made a few revisions to the checklist to address the points you raised at yesterday's meeting. This includes: 

- Making the construction of the interview guide as part of the overall development of a research plan. 
- I also broke the section on creating a summary template (step 4) into two sections (now steps 4 and 5). 
- And I clarified the question I had listed about whether or not to use a single worksheet within a spreadsheet workbook or multiple worksheets within a spreadsheet workbook. I just made this into an option at the end of step 7.

**Comment by dialectic51** [comment]
Update on RQA for RP001: I have created templates for both the [Summary Template](https://docs.google.com/document/d/19nwabYmVlpG0GKROYvx5ZVR7R8vKHA--1kmLWACFKWY/edit) and the [Matrix Analysis](https://docs.google.com/spreadsheets/d/1WT4zQXFvST1UygloeXcwa-AVm6x2o5c2U9Nky7Bd8qc/edit#gid=0) to be used once we start conducting the RQA (located in [this folder](https://drive.google.com/drive/folders/1hKEeL17224X_HHPe19D1go6-AEajr9Ij) of the Internship google drive).

They are based off of [slide #23](https://www.hsrd.research.va.gov/for_researchers/cyber_seminars/archives/780-notes.pdf#page=23) and [slide #33](https://www.hsrd.research.va.gov/for_researchers/cyber_seminars/archives/780-notes.pdf#page=33) of Hamilton’s presentation and used in steps #4 and #7 in the [RQA checklist](https://github.com/hackforla/internship/issues/338#issuecomment-1327682586). I have placed links to the templates in the instructions/resources section of [issue#338](https://github.com/hackforla/internship/issues/338). (Note: these are templates, but I haven’t been able to figure out how to turn them into actual templates in google docs and sheets).

The summary template was pretty straightforward from Hamilton’s slides. However, I made some changes to the Matrix in order to bring in two additional columns next to each domain: 1.) analytical memos and 2.) themes and trends. (Constructing analytical memos and identifying themes and trends are key steps in conducting a matrix analysis).

I have also placed in bullet points in each cell under the domain columns. The bullet points in the Matrix should be taken directly from the bullet points that will be created in the Summary Templates by a coding team. However, bullet points are a little wonky in Google Sheets, so I have a link to directions on how to create them at the bottom of the document.

(This note was also sent via Slack to Sara, Linda, and Angela).

**Comment by pandanista** [comment]
@dialectic51 Hi Brian! Welcome back from the break. We are confirming everyone's interest and availability.  Please provide a new update to this issue you have been working on. Thank you. 

Please provide update:
1. Progress: "What is the current status of your project? What have you completed and what is left to do?" - we know you just came back from break, please just summarize the progress from before then
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by dialectic51** [comment]
@pandanista 

1. Progress: I have finished the checklist for the RQA for RP001 ([see comment above dated 11/25](https://github.com/hackforla/internship/issues/338#issuecomment-1327682586)). I have also completed the templates needed for the RQA, specifically the "Summary Template" and "Matrix Analysis" documents ([see comment above dated 12/2](https://github.com/hackforla/internship/issues/338#issuecomment-1335822005)). I met with Sara on 12/14 and she helped me turn these into genuine Templates (in line with HackForLA standards), place them into the appropriate place on the google drive, and link them on the Wiki for RP001. The next steps will be to: 
-a.) create a Guide for conducting RQA based on the checklist I've created; 
-b.) break the different steps of the RQA checklist into issues (and create an Epic issue to organize them all); and 
-c.) conduct the RQA for RP001. 
-(Later on, the RQA checklist will be turned into a Template).   
3. Blockers: Once Angela returns from the break, she will look over the checklist and give me feedback if she has any. We will then begin the RQA with Angela serving as the UX lead.
4. Availability: As needed.
5. ETA: I'm hoping to complete the Guide (at least a rough draft) and create the Issues next week; we would then be able to begin the RQA in the week starting 1/16

**Comment by dialectic51** [comment]
@Dewey-Tran To prepare you for reviewing the [checklist,](https://github.com/hackforla/internship/issues/338#issuecomment-1327682586) look over [Linda's Presentation to UXR Team on How to Conduct Rapid Qualitative Analysis](https://docs.google.com/presentation/d/12j99CF8fnvdb7WOWxDn1_CO_UODjoL23HXZUqujbveA/edit?usp=sharing) and the [Transcript of Linda's Presentation on RQA](https://www.hsrd.research.va.gov/for_researchers/cyber_seminars/catalog/transcripts/780.doc), which are also located under the Information/Resources section in the Overview. These are the documents that I based my checklist on.

**Comment by Dewey-Tran** [comment]
I had reviewed the list @dialectic51 created, here are my suggestions of tasks to include. 

1.) Develop a Research Plan
- [ ] Ensure data collection procedure is an iterable process

2.) Collect data/conduct interviews
- [ ] Keep note of significant events interviews (

8.) Conduct matrix analysis
- [ ] Provide suggestions for future analyses

**Comment by pandanista** [comment]
@adarosh I am assigning you to review the checklist above. Let me know if you have any questions. Thank you!

**Comment by adarosh** [comment]
Hi @dialectic51 and @pandanista - please see below for feedback. Hope some of these suggestions help- if you have any questions about my comments, just let me know. Looks great so far!

**1. Develop a research plan:**
- Add “qualitative” to data type under determine if RQA is appropriate
- Define action oriented in parentheses
- Recruiting participants has a bullet for using explanatory deductive approach. Is this appropriate here?
- Under introductory script- you include purpose of the study, how data will be used, and participant confidentiality. We should lump this under informed consent/assent, which should also include study contact information, right to withdraw, etc. Add note that we need to have consent/assent prior to asking any questions.
- Next to “interview questions”- add that interview should be semi-structured or structured
- Under closing script- add a bullet thanking participants for their time and reminding them of study contact information
- Under data analysis plan- it gives options that are not RQA. I understand this for a general research plan, but if this is for specifically RQA does this need to be here? At minimum, add RQA to the options listed.
- Put an end parenthesis for the bullet that states “Estimate timeline for conducting the analysis…”
- Put together plan for training research team- add a bullet for “create training guide” and “schedule training session”

**2. Collect data/conduct interviews:**
- Not sure if this is complete, but there should be bullets about how the data are collected, how it will be stored, testing technology (if using Zoom, etc.)

**3. Transcribe and read interviews OR listen to recordings and take notes:**	
- This may need to be fleshed out more. Again, with how/where this will be done (e.g., Google forms, docs, etc.)

**4. Create a summary template for use by team in word document:**
- Change word document to Google doc

**5. Take summary template for “test drive”:**
- Make a note that these were taken from the slides and link to them (or add note with the slide number) so that people can refer back to the deck
- Maybe after first run through of this the checklist can be edited for ease of understanding as we get used to it

**6. Divide transcripts:**
- Add a bullet about tracking who has which transcripts to review

**7. Transfer summaries:**
- Not sure what is meant by developing a templated summary of each data collection episode. Is this within the matrix? An additional file? Is this different from step 6? I think this needs to be clarified
- Fill in data on a single worksheet -OR- break into multiple worksheets- note here that this is determined by the data team, not individual reviewers

**8. Conduct matrix analysis:**
- This may need to be edited or fleshed out after the first analysis
- Add a link or reference to training deck so researchers can refer back to training

**9. Identify themes within domains:**
- Add note about discussion and consensus of themes (since this should be reviewed by two or more analysts)
- This may require more guidance for how to identify themes

**10. Prepare report/presentation:**
- This may link to a Github/Google doc template or checklist when available

**Comment by dialectic51** [comment]
Currently updating RQA Checklist based on [the feedback provided by Ashley.](https://github.com/hackforla/internship/issues/338#issuecomment-1409403242) I have completed incorporating the suggestions for Step 1 (Develop a research plan). I should be able to finish incorporating the rest of the feedback either later today or tomorrow.

**Comment by dialectic51** [comment]
Finished incorporating the feedback provided by Angela into the RQA checklist. The RQA Checklist should be ready to be taken through an initial run to see how well it works, making additions or changes based on the experience. 

I will remove myself from the issue at our next meeting so that I can work on more time sensitive materials, such as RP020.

**Comment by pandanista** [comment]
- Need to assign a team member to revise the checklist above.

- [Here](https://github.com/hackforla/internship/issues/492#issuecomment-2103186587) is an example from RP017's checklist.

**Comment by jinyan0425** [comment]
@pandanista I reviewed all the comments and related issues. Before I start revising the checklist, I think I need to know more about the context and objective of this task. We can discuss this in today's UX meeting.

A minor issue is that I do not have access to the following folders/docs:
1. [Folder on UI/UX CoP Google Drive for Rapid Qualitative Analysis](https://drive.google.com/drive/folders/1d9xZF6MR5OXSrfX8d-MI1BVSZHxesg-A)
2. [UI/UX Guide for How to Conduct Rapid Qualitative Analysis](https://docs.google.com/document/d/1RjR7Oh9KZ2bQlJGy1MyNFwYjtxeDxawza9P6ViKOL64/edit?usp=share_link)

**Comment by pandanista** [comment]
Hi @jinyan0425, I just found the instructions to be added back to the UI/UX COP Google Drive. See detailed info [here](https://hackforla.slack.com/archives/C017ESHSMNG/p1715106432621009).

**Comment by jinyan0425** [comment]
> Hi @jinyan0425, I just found the instructions to be added back to the UI/UX COP Google Drive. See detailed info [here](https://hackforla.slack.com/archives/C017ESHSMNG/p1715106432621009).

@pandanista Thanks, Ying! Just filled out the form.

**Comment by jinyan0425** [comment]
@pandanista 
1. I read through the transcript, which is not the transcript of Linda's presentation but that of Hamilton's presentation back in 2013. So, the label of this resource is incorrect: [Linda's Presentation to UXR Team on How to Conduct Rapid Qualitative Analysis](https://docs.google.com/presentation/d/12j99CF8fnvdb7WOWxDn1_CO_UODjoL23HXZUqujbveA/edit?usp=sharing). We may need to correct it. On the other hand, it will be very useful if we can find the actual transcript for Linda's presentation.

2. **I would like to confirm we will fully follow Hamilton's Process/Approach, which relies on verbatim transcript instead of identifying themes directly from audio recordings**. Hamilton's Process/Approach is an early approach to RQA. See the following expert from a [recent method paper on RQA -- Nevedal et al. 2021](https://implementationscience.biomedcentral.com/articles/10.1186/s13012-021-01111-5). This paper was also cited in [Linda's presentation on pp.10](https://docs.google.com/presentation/d/12j99CF8fnvdb7WOWxDn1_CO_UODjoL23HXZUqujbveA/edit#slide=id.g18b90d5d4a8_0_49):

>  Hamilton developed a rapid qualitative analysis approach that summarizes transcript data into templates using domains aligned with interview questions; summary points are then distilled into a matrix organized by domain and participant for analysis and interpretation [[18](https://implementationscience.biomedcentral.com/articles/10.1186/s13012-021-01111-5#ref-CR18)].

> The rapid approaches described by Hamilton and Gale et al. rely on verbatim transcripts, which means teams must wait for transcription to be completed to proceed with rapid or traditional analyses. In contrast, Neal et al. [[28](https://implementationscience.biomedcentral.com/articles/10.1186/s13012-021-01111-5#ref-CR28)] developed an approach to rapidly identify themes directly from audio recordings.

**Comment by jinyan0425** [comment]
### Draft of Rapid Qualitative Analysis (RQA) Checklist
----
_@pandanista This is tentative and developing. I made comments if applicable after each point in italics_


**1. Get familiar with the Hamilton's Process of RQA**
- If you are new to RQA, review the following methodology documents to familiarize yourself with Hamilton's Process: 
    - **[MUST READ]** [Qualitative Methods in Rapid Turn-Around Health Services Research](https://www.hsrd.research.va.gov/for_researchers/cyber_seminars/archives/video_archive.cfm?SessionID=780)
    - [Linda's Presentation to UXR Team on How to Conduct Rapid Qualitative Analysis](https://docs.google.com/presentation/d/12j99CF8fnvdb7WOWxDn1_CO_UODjoL23HXZUqujbveA/edit?usp=sharing)
- If you are experienced with RQA, you may skip reviewing the documents or review them for a refresher.

_COMMENTS: Many researchers may not be familiar with RQA, especially Hamilton's Process. Therefore, I think the first step should be to become familiar with this method. Additionally, I do NOT want to mention [Nevedal et al., 2021](https://implementationscience.biomedcentral.com/articles/10.1186/s13012-021-01111-5), which could potentially confuse researchers if we exclusively focus on Hamilton's Process (see my comment above)._


**2. **

**Comment by pandanista** [comment]
> @pandanista
> 
> 1. I read through the transcript, which is not the transcript of Linda's presentation but that of [Hamilton's presentation back in 2013](https://www.hsrd.research.va.gov/for_researchers/cyber_seminars/archives/video_archive.cfm?SessionID=780). So, the label of this resource is incorrect: [Linda's Presentation to UXR Team on How to Conduct Rapid Qualitative Analysis](https://docs.google.com/presentation/d/12j99CF8fnvdb7WOWxDn1_CO_UODjoL23HXZUqujbveA/edit?usp=sharing). We may need to correct it. On the other hand, it will be very useful if we can find the actual transcript for Linda's presentation.
- I don't think we have the actual transcript from Linda's presentation because it was not recorded. 
- I am not sure which label is incorrect. Would you please point me to it?
    -  Hamilton's work is only in the references of Linda's presentation, but we can list out the source too.

> 2. **I would like to confirm we will fully follow Hamilton's Process/Approach, which relies on verbatim transcript instead of identifying themes directly from audio recordings**. Hamilton's Process/Approach is an early approach to RQA. See the following expert from a [recent method paper on RQA -- Nevedal et al. 2021](https://implementationscience.biomedcentral.com/articles/10.1186/s13012-021-01111-5). This paper was also cited in [Linda's presentation on pp.10](https://docs.google.com/presentation/d/12j99CF8fnvdb7WOWxDn1_CO_UODjoL23HXZUqujbveA/edit#slide=id.g18b90d5d4a8_0_49):
> 
> > Hamilton developed a rapid qualitative analysis approach that summarizes transcript data into templates using domains aligned with interview questions; summary points are then distilled into a matrix organized by domain and participant for analysis and interpretation [[18](https://implementationscience.biomedcentral.com/articles/10.1186/s13012-021-01111-5#ref-CR18)].
> 
> > The rapid approaches described by Hamilton and Gale et al. rely on verbatim transcripts, which means teams must wait for transcription to be completed to proceed with rapid or traditional analyses. In contrast, Neal et al. [[28](https://implementationscience.biomedcentral.com/articles/10.1186/s13012-021-01111-5#ref-CR28)] developed an approach to rapidly identify themes directly from audio recordings.
- Since our transcripts are mostly ready, I think we could follow Hamilton's Process/Approach. What are the benefits for Nevedal's approach compared to Hamilton's?

**Comment by pandanista** [comment]
COMMENTS regarding [RQA's updated checklist](https://github.com/hackforla/internship/issues/338#issuecomment-2192408993):

- Many researchers may not be familiar with RQA, especially Hamilton's Process. Therefore, I think the Step should be to become familiar with this method. In Step 1, I do NOT want to mention [Nevedal et al., 2021](https://implementationscience.biomedcentral.com/articles/10.1186/s13012-021-01111-5), which could potentially confuse researchers if we exclusively focus on Hamilton's Process (see my comment above).
    - I agree if we follow Hamilton's approach. 
- Steps 2 and 3 can be combined as Conduct Transcript Summaries.
    - It might be easier to leave them separated as step 2 is for set-up and testing and step 3 is for summarizing all transcripts. 
    - From the operational standpoint, not every assignee needs to create and set up the summary template, etc.  
    - Here is what I am thinking regarding the process: The assignee who got assigned to work on the analysis first, they need to create a summary template (maybe a matrix too) and do a round of test. Then, another team member can do a peer review. Lastly, we use the updated template/checklist and distribute to the team so other team members can work on the summary table and matrix.
        - Looks very similar to what you drafted. Let me know your thoughts as well. 
- The sub-points in each step may still be too detailed. If so, we can remove some of them.
On the other hand, RQA is quite complex, with many nuances even at each lowest-level step in the checklist. For instance, there are detailed instructions on how to "Compare the summarizing 'styles' across the team". If we remove this bullet point from the checklist, we may still need to reintroduce all these details as action items in the actual data analysis phase/issue (e.g., https://github.com/hackforla/internship/issues/493)
    - Good point. Maybe we should leave them as detailed for now. I wonder if we could remove the detailed parts and have all the detailed information in the RQA guide that will be based on our initial set of analyses (RP001). But for our initial set of analyses (RP001), we don't have the guide ready yet.
- Another challenge is the high flexibility of RQA, where many steps need to be customized for specific RPs. This requires careful construction of the actual data analysis action items tailored to each RP.
    - What kind of customization are your referring to?

**Comment by pandanista** [comment]
@sunannie27 
- Please review the [checklist](https://github.com/hackforla/internship/issues/338#issuecomment-2192408993). Let me know if you have any questions.

- If the checklist looks good, the research leads will use that to make RP001's data analysis new issue.

**Comment by KC-skc** [comment]
(not transcripts_) Is this a typo? Are you trying to italicize as well as bold?

**Comment by jinyan0425** [comment]
> (not transcripts_) Is this a typo? Are you trying to italicize as well as bold?

YES, KC! Thanks for catching it. Edited.

**Comment by sunannie27** [comment]
@jinyan0425 @pandanista Checklist looks good. Will kickback to research leads.

**Comment by pandanista** [comment]
- Closing this issue since it is signed off and we copied the check list to the data analysis issue #616.
